weave/trace_server/current_calls_query.sql

```diff
@@ -0,0 +1,192 @@
+-- This is the current call stream query
```

Comment:
this is just a copy of the query executed on the trace page for more readability